src/mtype.c

```diff
@@ -5738,6 +5738,27 @@ void TypeFunction::purityLevel()
     }
 }
 
+MATCH TypeFunction::modMatch(Type *tthis)
```

Comment:
It does mostly same work with `MODmethodConv`. I'll improve it rather than adding a new function.